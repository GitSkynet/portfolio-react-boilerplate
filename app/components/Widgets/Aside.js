import styled from 'styled-components';

const Aside = styled.aside`
  align-self: flex-start;
  display: flex;
  margin-left: auto;
  background-color: #fff;
  color: #575756;

  > aside:not(:last-of-type) {
    border-right: 1px solid #575756;
  }

  @media (min-width: 480px) {
    padding: 0 1em 0 4em;
    background-color: transparent;
    background-image: linear-gradient(45deg, transparent 17%, #fff 17%);
    background-position-x: 0.5em;
  }
`;

export default Aside;
