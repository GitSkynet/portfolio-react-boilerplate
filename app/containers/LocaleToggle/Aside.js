import styled from 'styled-components';

const Aside = styled.aside`
  box-shadow: 0 0 10px grey;
  opacity: 0.9;
  transition:
    box-shadow 0.3s 0s ease-in-out,
    opacity 0.3s 0s ease-in-out;

  &:focus,
  &:hover {
    box-shadow: 0 0 5px lightgrey;
    opacity: 1;
  }
`;

export default Aside;
