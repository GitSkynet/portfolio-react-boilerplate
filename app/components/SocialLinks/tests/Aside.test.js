import React from 'react';
import renderer from 'react-test-renderer';

import Aside from '../Aside';

describe('Aside', () => {
  it('Should render and match the snapshot', () => {
    const tree = renderer.create(<Aside />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
