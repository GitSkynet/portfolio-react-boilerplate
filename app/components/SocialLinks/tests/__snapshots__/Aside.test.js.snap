// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Aside Should render and match the snapshot 1`] = `
<aside
  className="Aside__Aside-dWuibx AChCr"
/>
`;
